src/components/vm/consoles/vnc.jsx

```diff
@@ -377,6 +393,17 @@ export class VncActive extends React.Component {
         return window.location.protocol === 'https:';
     }
 
+    onConnected(vnc_element) {
+        let canvas = null;
+        if (this.props.onRemoteSizeChanged)
+            canvas = vnc_element.querySelector("canvas");
+        if (canvas) {
+            this.observer.observe(canvas, { attributes: true });
+            this.report_remote_size(canvas);
```

Comment:
is this `report_remote_size` call here necessary? If I'm reading the `MutationObserver` docs right after you call `.observe()` it will use the callback method which also calls `report_remote_size` method.

Reply:
Aha, hmm. I added it here for Chrome, when it started failing a test. But I didn't investigate carefully. Will do!

Reply:
But: I have added a check that skips the window resizing when the onRemoteResizeChanged callback is called, but the the size hasn't actually changed.